technical_overview/technical_details/Interchange_format.md

```diff
@@ -0,0 +1,47 @@
+# The Interchange format as basis for data integration
+
+## Background
+As detailed in the illustration of the data flow, the EMAP pipeline has the potential to consolidates data from several
+sources and more can be added over time as and when is necessary. For the moment, the data is provided through two 
+different sources: 
+    
+1. incremental load from databases (also referred to as hoovering) and
+1. HL7 message streams
+
+Due to the different sources, the data provided to the EMAP pipeline differs in the way it is formatted and it is 
+necessary to define a common a structure for the purpose of consolidation. While both the  
+[HL7 standard](https://www.hl7.org/implement/standards/) and the [FHIR standard](https://www.hl7.org/fhir/overview.html)
+were considered for this purpose, the aim behind both of these standards is that a digital conversation between the 
+different technologies used in the hospital can be enabled, but not so much the integration of all the different data 
+sources. Therefore, it was decided to introduce the Interchange format that is less variable and more conducive to data
+integration. 
+
+
+## Key differences between HL7 and Interchange format
+
+The first key difference between both the formats is that all Interchange messages have detailed information about the 
```

Comment:
hl7 messages have more provenance information most of the time, but they just aren't used consistently 
```suggestion
The first key difference between both the formats is that all Interchange messages have consistent  use of fields for the 
```